src/back/router.py

```diff
@@ -0,0 +1,9 @@
+from typing import Self
+from tornado.iostream import IOStream
+
+
+class Router(MessageHandlerInterface):
+    """"""
+
+    def handle(self: Self, message: bytes, stream: IOStream) -> None:
```

Comment:
I like that the router is just another handler. Just make all the changes that you would do in the interface.